### ISSUE TITLE
Wrong Unicode mappings in masters

### ISSUE THREAD

**Issue by jonpinhorn** [itfoundry/hind-jalandhar#1]
It looks like there's a mistake, in the Glyphs file, at least:

grYakash: 0A51
grUdaat: 0A75

Shouldn't be the other way around?

**Comment by lianghai** [comment]
Yes it's a mistake. GOADB is good so this doesn't affect compiled fonts.
Anyway, will fix it:

```
grYakash 0A51 => 0A75
grUdaat 0A75 => 0A51
```

**Comment by davelab6** [comment]
@lianghai is this fixed?
